lessons/commutative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,116 @@
+* Demonstrate their understanding of the Commutative Property via transformations of Circles, numeric expressions, and words.
+
+| Student-facing Lesson Goals
+|
+
+* Let's explore the Commutative Property and Circles of Evaluation.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* Pay attention to students' Circle Transformations to ensure that they are in fact applying the Commutative Property. Students will likely "accidentally" discover the Associative Property today; feel free to acknowledge the equivalence of Circle Transformations involving changes in grouping, but (for now) keep students focused on Transformations that only affect the order of the values.
+|===
+
+== The Commutative Property
+
+=== Overview
+
+Students analyze pairs of Circles to see if they represent equivalent expressions. The Circles make visible the structural changes that occur when we apply the Commutative Property.
+
+=== Launch
+
+Build on prior knowledge by reminding students of the activity that they completed in the previous lesson. You might even refer to the list of category names that the class created, and then draw students’ attention to any category names that allude to or suggest the commutativity. (Any mention of “changing the order” is a strong hint that students’ have clued into the Commutative Property!)
+
+[.lesson-instruction]
+- Now, turn to Discover the Commutative Property. Look at the first problem. What do you Notice about the two Circles? What do you Wonder?
+- Work through the remaining problems with your partner. Be sure to evaluate the circles to confirm if they produce equivalent outcomes.
+- What did you observe about the Commutative Property?
+** _Possible responses: In each of the examples, the values were the same but the order of the values was different. Sometimes the Circle structure changed, like when I moved a nested circle on the right to become a nested circle on the left. The Commutative Property holds for multiplication and addition, but not subtraction or division._
+
+=== Investigate
+
+[.lesson-instruction]
+- Turn to Commutative Property Table.
+- How many different ways can you apply the Commutative Property? Draw as many different Circles of Evaluation as you can.
+
+This activity is similar to one students’ completed during the Computation as a Circle Transformation lesson.  In both activities, students are developing equivalent expressions. That said, in this lesson, students cannot write down any equivalent circle; they must develop Circle Transformations that apply the Commutative Property.
+
+Note that each of these Circles can be represented in at least one alternative way. Some circles, however, include subtraction and division. You may need to draw some students' attention to the operation as they rearrange the values inside the circle, helping them to notice that the Commutative Property does not apply for division and subtraction.
```

Comment:
An extension of this could involve negative numbers.
Whereas 5 - 6 does not equal 6 - 5, 5 - 6 is equivalent to 5 + -6 which does equal -6 + 5. And this is sixth grade work.

Similarly, for division it might be worth making the connection to fractions.
8 / 2 does not equal 2/ 8 but 8 /2 can be written as 8 * 1/2, which is equivalent to 1/2 * 8